xrootd4j-gsi/src/main/java/org/dcache/xrootd/plugins/authn/gsi/GSIAuthenticationHandler.java

```diff
@@ -245,19 +245,19 @@ public int getSize() {
                                               responseBuckets.getBuckets());
         } catch (InvalidKeyException ikex) {
             _logger.error("Configured host-key could not be used for" +
-                          "signing rtag: {}", ikex);
+                          "signing rtag: {}", ikex.getMessage());
             throw new XrootdException(kXR_ServerError,
                                       "Internal error occurred when trying " +
                                       "to sign client authentication tag.");
         } catch (CertificateEncodingException cee) {
             _logger.error("Could not extract contents of server certificate:" +
-                          " {}", cee);
+                          " {}", cee.getMessage());
             throw new XrootdException(kXR_ServerError,
                                       "Internal error occurred when trying " +
                                       "to send server certificate.");
         } catch (IOException | GeneralSecurityException gssex) {
             _logger.error("Problems during signing of client authN tag " +
-                          "(algorithm {}): {}", SERVER_ASYNC_CIPHER_MODE, gssex);
```

Comment:
Note that, in many cases, OpenJDK sends an IOException without any message.  Presumably the class name is meant to be sufficiently descriptive.

Therefore, simply logging the message when the caught exception could be an IOException isn't a great choice